### ISSUE TITLE
"go to" document for Sphinx installation.

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1039]
Hi, I'm a developer for the LLVM project and we have run into a need for more comprehensive documentation regarding how to properly install Sphinx.

The current state of Sphinx's documentation regarding installation appears to be written for an audience of Python developers (not surprising, given Sphinx's origins and, I'm sure, most current users). However, in the LLVM project, we are primarily C++ developers and hence "just easy_install it" is inadequate installation guidance (unless, like me, you also happen to have a background in Python).

Currently, as part of the LLVM project, we have a small Sphinx intro http://lld.llvm.org/sphinx_intro.html maintained in one of the subprojects (LLD) which gives some guidance on this front (among other things), but I would like to have something "official" that I can point to. As a knee-jerk reaction I was going to write a document of this nature myself (to the extent of my abilities) and put it in the LLVM docs, but after taking a step back, I realized that this information really should be provided upstream as part of Sphinx's own documentation.

So, basically, what we need is a document regarding setup of Sphinx that:
- Assumes zero knowledge of Python (doesn't assume the reader knows what easy_install is).
- Provides step-by-step instructions for how to install on at least Linux, Mac, Windows. Nothing fancy, but please be clear and thorough.
- [optional] how to set it up in a virtualenv (I just say this because our existing intro includes it; I don't consider it _that_ important, but it would be nice)

I'm not too familiar with Sphinx's layout, but I imagine this being a top-level .rst file sitting in doc/, with a title like "Installing Sphinx" or "Getting Sphinx".

tl;dr: I want a single document, maintained by the Sphinx developers, that I can point to for "how to install Sphinx"; it should assume that readers are not necessarily Python developers (e.g. don't know what easy_install is).

As always, feedback is welcome and appreciated. Thanks for your time, and thank you for Sphinx!

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1039
- Originally reported by: Sean Silva
- Originally created at: 2012-11-14T05:08:16.467

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-11-19 00:52:34+00:00_

sphinx-users.jp wrote some startup document for sphinx in Japanese (http://sphinx-users.jp/gettingstarted/index.html ). I might be good if this is useful, then translated into English, be incorporated into the official documents.

Sample English version is here (temporary):
- html: http://sphinxjp.bitbucket.org/gettingstarted_e/index.html
- repo: https://bitbucket.org/sphinxjp/website_e

**Comment by shimizukawa** [comment]
_From Claudia McGhee on 2012-12-12 22:09:52+00:00_

Yes, this information on how to install on different operating systems is sorely needed by those of us new to Sphinx, and to those like me, who work in Windows!  I have written install/config guides for sys admins in Windows environments, so I can certainly help once I know what needs to be done!

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 18:56:46+00:00_

I think enough of that info has been included now: http://sphinx-doc.org/install.html
